code/modules/antagonists/traitor/objectives/locate_weakpoint.dm

```diff
@@ -110,29 +111,12 @@
 	weakpoint_found = TRUE
 
 /datum/traitor_objective/locate_weakpoint/proc/create_shockwave(center_x, center_y, center_z)
-	var/severity = list(EXPLODE_LIGHT, EXPLODE_LIGHT, EXPLODE_LIGHT, EXPLODE_LIGHT, EXPLODE_HEAVY, EXPLODE_HEAVY, EXPLODE_DEVASTATE) //Can't use pick_weight because explode defines are numbers
-	var/wave_amount = rand(5, 8)
-	var/list/bombed_turfs = list()
-	for(var/i in 1 to wave_amount)
-		var/wave_angle = rand(-10, 10) + 360 / wave_amount * i
-		var/wave_distance = rand(17, 25)
-		var/turf/tentacle_ending = locate(clamp(center_x + round(cos(wave_angle) * wave_distance), 1, world.maxx), clamp(center_y + round(sin(wave_angle) * wave_distance), 1, world.maxy), center_z)
-		if(!tentacle_ending) //WUT
-			continue
-
-		var/turf/epicenter = locate(center_x, center_y, center_z)
-		for(var/turf/line_turf in get_line(epicenter, tentacle_ending))
-			for(var/turf/bomb_turf in range(1, line_turf))
-				if((bomb_turf in bombed_turfs) || bomb_turf == epicenter)
-					continue
-				bombed_turfs += bomb_turf
-				var/turf_severity = pick(severity)
-				EX_ACT(line_turf, turf_severity)
-				for(var/atom/victim in line_turf)
-					EX_ACT(victim, turf_severity - 1)
-
-		explosion(tentacle_ending, devastation_range = 1, heavy_impact_range = 3, light_impact_range = 5, explosion_cause = src)
-
+	var/turf/epicenter = locate(center_x, center_y, center_z)
+	var/lowpop = (length(GLOB.clients) <= 20)
+	if(lowpop)
```

Comment:
Remove the var, and put the condition in here directly.

Next, make the 20 a define so we can more easily change it later. Might even be the type of thing to config since lowpop is different from server to server, but I wouldn't mind if you didn't

Reply:
:+1: on this, lowpop isn't the same on every server so I'd recommend making it a config too, or at the very least an easy-to-find define.

Reply:
Adjusted to use CONFIG_GET(number/minimal_access_threshold)